ui/de-lib/src/main/java/org/iplantc/de/client/sharing/SharingPresenter.java

```diff
@@ -0,0 +1,20 @@
+package org.iplantc.de.client.sharing;
```

Comment:
I'd prefer that this code be kept in `org.iplantc.de.sharing.client`. Just because the package has this pattern does not mean that the code underneath it *MUST* be a GWT module. If we end up having more sharing views, widgets, presenters, etc which need to be used across the app, it would make more sense for it to be in a namespace which matches the rest of our app.